src/index.js

```diff
@@ -17,6 +17,9 @@ import {getPlayers, getPlayer} from './proxy';
 import {cast} from './common/cast';
 // Import playlist
 import {playlist} from './common/playlist';
+import 'intersection-observer';
```

Comment:
as we are not re-exporting it, why not import it from the manager who is responsible for it.
In the future if we add dynamic loading then we will just need to refactor the manager.

Reply:
done